plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -372,7 +372,9 @@ function perflab_install_and_activate_plugin( string $plugin_slug, array &$proce
 		// Replace new Plugin_Installer_Skin with new Quiet_Upgrader_Skin when output needs to be suppressed.
 		$skin     = new WP_Ajax_Upgrader_Skin( array( 'api' => $plugin_data ) );
 		$upgrader = new Plugin_Upgrader( $skin );
-		$result   = $upgrader->install( $plugin_data['download_link'] );
+		// Remove the version number from the link to download the latest plugin version.
+		$download_link = (string) preg_replace( '#(\/plugin\/[^\/]+)\.\d+\.\d+\.\d+\.zip#', '$1.zip', $plugin_data['download_link'] );
```

Comment:
Thank you for pointing this out! I hadn’t considered the possibility of version tags like `-rc` or `-beta`. That’s an oversight on my part.

However, I believe the regex `'#(\/plugin\/[^\/]+)\.\d.*.zip#'` might not work as intended because it could capture the plugin name greedily. For instance, in the example provided, the download link might end up being `https://downloads.wordpress.org/plugin/rest-api.2.zip` instead of `https://downloads.wordpress.org/plugin/rest-api.zip`.

To address this, one solution could be to modify the regex to capture the plugin name non-greedily, like this:

```php
'#(\/plugin\/[^\/]+?)\.\d.*.zip#'
```

Alternatively, another approach would be to use the following regex:

```php
'#(\/plugin\/[^\/]+)\.\d+\.\d+\.\d+(?:-\w+)?\.zip#'
```

This pattern accounts for optional tags like `-rc` or `-beta` and aligns with the regex used in the `versions` command, so it seems like a good choice to me.